frontend/src/metabase/visualizations/echarts/cartesian/model/axis.ts

```diff
@@ -525,6 +530,7 @@ export function getYAxesModels(
   columnByDataKey: Record<DataKey, DatasetColumn>,
   isAutoSplitSupported: boolean,
   stackModels: StackModel[],
+  compactSeriesByDataKey: DataKey[],
```

Comment:
Nit: I'd call this `compactSeriesDataKeys` because "a by y" is frequently used for dictionaries/maps